### DISCUSSION TITLE
Java?

### DISCUSSION THREAD

**Discussion by lordtgm** [TrinityCore/TrinityCore#28916] [Suggestions]
Well this is one of those really hard and time demanding requests, but why not move to java or at least have a java version? its actually more modern and most importantly easier to both code and compile.

**Comment by Shauren** [comment]
Sure, let me rewrite a million line of code project to another language

**Reply by hwis** [reply]
if you decide to rewrite, write in javascript first)))

**Comment by Aokromes** [comment]
go https://github.com/CypherCore/CypherCore
